### PR TITLE
Feat: fuzz testing

### PR DESCRIPTION
**Pull request by 678098**
Disabled PUT message fuzzing since fixing it might take more time

### CODE REVIEW

File: src/python/blazingmq/schemas/broker.py [hunk 1]
```diff
@@ -1,5 +1,5 @@
 """
-BMQ Broker schemas.
+BlazingMQ Broker schemas.
```
Comment (pniedzielski) [inline src/python/blazingmq/schemas/broker.py:2]: 👍🏻

File: src/python/blazingmq/dev/fuzztest/__init__.py [hunk 2]
```diff
@@ -0,0 +1,406 @@
+"""
```
Comment (pniedzielski) [inline src/python/blazingmq/dev/fuzztest/__init__.py:1]: This is unchanged from the previous PR, right?  Except for the pre-existing typo?

File: .github/workflows/build.yaml [hunk 3]
```diff
@@ -175,6 +175,30 @@ jobs:
...
+    name: Fuzz test [${{ matrix.request }}]
+    strategy:
+      matrix:
+        request: ["identity", "open_queue", "configure_queue_stream", "confirm", "close_queue", "disconnect"]
+      fail-fast: false
+    runs-on: ubuntu-latest
+    needs: build_ubuntu
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/cache/@v4
```
Comment (pniedzielski) [inline .github/workflows/build.yaml:188]: Caching pays dividends again!

**Review by pniedzielski — APPROVED** [review]
Looks good!

File: src/python/blazingmq/dev/fuzztest/__init__.py (hunk 2 above)
Comment (678098) [reply src/python/blazingmq/dev/fuzztest/__init__.py:1]: Yep

**Pull request merged**